source/common/hot_restart/BUILD

```diff
@@ -0,0 +1,35 @@
+licenses(["notice"])  # Apache 2
```

Comment:
@zuercher I feel really, really bad asking this, and I should have noticed before, but IMO the hot restart files should actually go in source/server, with namespace moved back to Server. This is so that we don't reach from common code back into server which we have been not doing so far (like for options). If you don't want to do this I can clean it up later. Sorry.